### DISCUSSION TITLE
Unable to capture TLS traffic of Envoy

### DISCUSSION THREAD

**Discussion by therealak12** [gojue/ecapture#633] [Q&A] [answered]
Hi.
I've tried to follow the same approach as in #367 but it doesn't work for me. Can anyone please help me find the cause? I'm trying to capture the ingress TLS traffic of an Envoy instance.

Ecapture version: 0.8.6
Envoy version: v1.31.0

![image](https://github.com/user-attachments/assets/d83d6434-14d0-49c7-9a41-64894a5a26f6)

**Comment by cfc4n** [comment, marked as answer]
It appears that the `SSL_Write` symbol is missing from Envoy's symbol table, likely due to the removal of debugging information during compilation. You will need to analyze Envoy yourself, locate the offset address of the `SSL_Write` symbol in the binary, and then recompile.

refer: #585

**Reply by therealak12** [reply]
Envoy publishes a `-debug` image per release. I'll try it. Thanks for the guidance.
